installing/installing_with_agent_based_installer/installing-with-agent-based-installer.adoc

```diff
@@ -31,11 +31,19 @@ include::modules/installing-ocp-agent-inputs.adoc[leveloffset=+2]
 
 You can create additional manifests to further configure your cluster beyond the configurations available in the `install-config.yaml` and `agent-config.yaml` files.
 
-If you do not want to make additional configurations, proceed to xref:../../installing/installing_with_agent_based_installer/installing-with-agent-based-installer.adoc#installing-ocp-agent-boot_installing-with-agent-based-installer[Creating and booting the agent image].
+If you do not want to make additional configurations, proceed to xref:../../installing/installing_with_agent_based_installer/installing-with-agent-based-installer.adoc#installing-ocp-agent-encrypt_installing-with-agent-based-installer[Optional: Encrypting the disk].
```

Comment:
It feels a bit weird that we are directing the user to an optional step, but I'm not sure what a better flow would look like.

Reply:
Yeah my first instinct was to put "proceed to the next section", though I wasn't sure whether users would misinterpret that and go to the disk partitioning section. I think I'll try "next section"